Framework/PythonInterface/test/python/plugins/algorithms/IndirectResolutionTest.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import (absolute_import, division, print_function)
+
+import unittest
+import mantid
+from mantid.simpleapi import IndirectResolution
+
+class IndirectResolutionTest(unittest.TestCase):
+    
+    def test_simple(self):
+        res_ws = IndirectResolution(InputFiles='IRS26173.raw',
+                                Instrument='IRIS',
+                                Analyser='graphite',
+                                Reflection='002',
+                                DetectorRange=[3, 53],
+                                BackgroundRange=[-0.16, -0.14],
+                                RebinParam='-0.175, 0.002, 0.175')
+                                
+        self.assertEqual(res_ws.getNumberHistograms(), 1)
+        self.assertEqual(res_ws.blocksize(), 175)
+        run = res_ws.run()
+        self.assertEqual(run.getProperty('res_back_end').value, -0.14)
+        self.assertEqual(run.getProperty('rebin_high').value, 0.175)
+        self.assertEqual(run.getProperty('rebin_width').value, 0.002)
+        
+    def test_scale(self):
+        res_ws = IndirectResolution(InputFiles='IRS26173.raw',
+                                Instrument='IRIS',
+                                Analyser='graphite',
+                                Reflection='002',
+                                DetectorRange=[3, 53],
+                                BackgroundRange=[-0.16, -0.14],
+                                RebinParam='-0.175,0.002,0.175',
+                                ScaleFactor=10)
+                                
+        self.assertEqual(res_ws.getNumberHistograms(), 1)
+        self.assertEqual(res_ws.blocksize(), 175)
+        self.assertEqual(res_ws.run().getProperty('res_scale_factor').value, 10)
```

Comment:
This is a good check to see that the scale factor has been added to the sample logs. I think it might also be worth checking that the scale has actually been applied to the workspace. You could do this by checking a bin matches an expected value or (even better) checking the scale factor vs. a non scaled workspace.